src/v2/Utils/SiftContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import sharify from "sharify"
+const sd = sharify.data
+
+export class SiftContainer extends React.Component {
+  componentDidMount() {
+    if (sd.THIRD_PARTIES_DISABLED || !sd.SIFT_BEACON_KEY) {
+      return
+    }
+
+    if (document.getElementById("sift-beacon")) {
+      return
+    }
+    const script = document.createElement("script")
+    script.id = "sift-beacon"
+    script.async = true
```

Comment:
I'm not sure `async` or `defer` do anything for inline text. Still, it's safe to leave it.

Reply:
This reminds me of a follow-up question: should we set `async` or `defer` on the injected `script` element?